### PR TITLE
Feature/removeexclusion

### PR DESCRIPTION
**Pull request by gokulavasan**
Removing Exclusion that is not required. 
Updating top level README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
-tigon-apps
-==========
+Tigon Applications
+==================
 
-Repository for reference applications for Tigon
+Repository of reference applications built using [Tigon](http://tigon.io/).
+
+[AdNetworkFlow](/AdNetworkFlow/): Realtime Bidding Advertisement Framework.
```
Comment (ParthGandhi) [inline README.md:6]: Should be "Real-time"

**Comment by ParthGandhi** [comment]
One comment. 

Fix and merge.

File: README.md (hunk 1 above)
Comment (awholegunch) [inline README.md:6]: We've been doing "realtime" elsewhere...

Comment (ParthGandhi) [inline README.md:6]: I used "real-time" in the app's README based on Achal's suggestion. 

I don't particularly care which one we use, as long as its consistent.

**Comment by gokulavasan** [comment]
Changed the README in AdNeworkFlow to realtime. @ParthGandhi take a look.

**Comment by ParthGandhi** [comment]
LGTM.

**Pull request merged**